src/homework4/task7.py

```diff
@@ -0,0 +1,18 @@
+"""
+    Task 7. Оглянемся назад. Числа.
+
+    Даны два натуральных числа. Вычислите их наибольший общий делитель при
+    помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+
+num1 = 12
+num2 = 0
+
+while num1 and num2:
+    if num1 > num2:
+        num1 = num1 % num2
+    elif num1 < num2:
+        num2 = num2 % num1
+
+print(f"Наибольший общий делитель - {max(num1, num2)}")
```

Comment:
It's a cheat - you just don't know where is result (second var is 0).
There is solution without this crutch